README.md

```diff
@@ -1,21 +1,66 @@
-# keylime
+# Keylime
 
 [![Build
 Status](https://travis-ci.org/keylime/keylime.svg?branch=master)](https://travis-ci.org/keylime/keylime) [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/keylime-project/community)
 
 ![keylime](doc/keylime.png?raw=true "Title")
 
-keylime is a scalable trusted cloud key management system. keylime provides an
-end-to-end solution for both bootstrapping hardware rooted cryptographic
-identities for IaaS nodes and for system integrity monitoring of those nodes via
-periodic attestation.
+Keylime is an open source scalable cloud trust system harnessing TPM Technology.
 
-keylime supports both TPM versions 1.2 and 2.0.
+Keylime provides an end-to-end solution for bootstrapping hardware rooted
+cryptographic trust for remote machines, the provisioning of encrypted payloads
+and run-time system integrity monitoring. It also provides a flexible
+framework for the remote attestation of any given `PCR` (Platform Configuration
+Register). Users can create their own customized actions that will trigger when
+a machine fails its attested measurements.
 
-keylime can be used with a software TPM emulator
-for development, testing, or demonstration purposes.  However, DO NOT USE keylime in
-production with a TPM emulator!  A software TPM emulator does not provide a hardware root
-of trust and dramatically lowers the security benefits of using keylime.
+Keylime's mission is to make TPM Technology easily accessible to developers and
+users alike, without the need for a deep understanding of the lower levels of a
+TPM's operations. Amongst many scenarios, it well suited to tenants who need to
+remotely attest machines not under their own full control (such as a consumer of
+hybrid cloud.)
+
+Keylime can be driven with a CLI application, web front end and a set of
+RESTful APIs.
+
+Keylime consists of three main components; The verifier, registrar and the
+agent. The verifier computes the measurements of the cryptographic TPM
```

Comment:
This sentence might be a bit jargon-y for the first few paragraphs of the README: "The verifier computes the measurements of the cryptographic TPM quotes accessed by the agents communication with the hosts TPM chip."

Maybe we can make this something more along the lines of "The verifier continuously verifies the integrity state of the machine that the agent is running on." ?

Reply:
Will use you change, sounds good to me.